### ISSUE TITLE
Machine Settings... window doesn't appear

### ISSUE THREAD

**Issue by dmopalmer** [daid/LegacyCura#770]
Configuration: Mac (OSX 10.9.2 Mavericks) In the latest build (SteamEngine branch, I don't know whether that is what I should be using)

When I open Machine Settings... the window doesn't appear.  However, when I Open Profile.. or Load Model, the Machine Settings... window appears at the same time as the file picker, and can be used if the file picker is dismissed.

Possible source of the bug, or maybe unrelated:
Putting Cura in the background and returning, the console gives the error message:
3/6/14 9:25:07.331 PM Cura[5714]: TypeError: MacReopenApp() takes exactly 2 arguments (1 given)

The wxPython documentation says that the signature should be
MacReopenApp(self)
But https://github.com/daid/Cura/blob/SteamEngine/Cura/gui/app.py has 
def MacReopenApp(self, event):
so the second argument in the def should be deleted.

**Comment by daid** [comment]
Did you build Cura yourself or did you use the release I build? (Due to the large amount of issues with wxWidgets on MacOS, this might be related to a self build)

**Comment by dmopalmer** [comment]
This was built from scratch, with a clone of the SteamEngine branch from Wednesday, following the (excellent) README instructions with no problems.

However, the 14.01 release has the same non-display of the Machine Settings.. dialog.  
(It does not have the MacReopenApp console error, which blame shows as coming from 
2014-02-13.)

To rule out it being a configuration file problem, I moved away my ~/.cura directory and started fresh (with the 14.01 release).  It went through the setup dialogs and brought up the main window with 'Welcome to the new version'  in front of it.  I closed the 'Welcome...' dialog, and tried to open machine settings... .  The dialog box did not appear.  Moving away the main window, I see that behind the main window was another 'Welcome...' dialog box, but with no content.  (The tiltle bar is Welcome... and the window is the same size and shape as it was before, but the content area is empty grey.)

Bringing up the model load picker makes:
1) the file picker pops up as it is supposed to
2) the empty welcome... box goes away
3) the machine settings... dialog comes up, layered behind the main window.
